furious/async.py

```diff
@@ -91,6 +91,7 @@ def run_me(*args, **kwargs):
 MAX_DEPTH = 100
 MAX_RESTARTS = 10
 DISABLE_RECURSION_CHECK = -1
+RETRY_SLEEP_SECS = 4
```

Comment:
I had considered that, but it did seem like overkill.  I think I'd like to leave as is for now.